landoapi/workers/base.py

```diff
@@ -152,7 +152,6 @@ def refresh_enabled_repos(self):
             for r in self.applicable_repos
             if treestatus_subsystem.client.is_open(repo_clone_subsystem.repos[r].tree)
         ]
-        logger.info(f"{len(self.enabled_repos)} enabled repos: {self.enabled_repos}")
```

Comment:
Yes, I am not sure if it is a useful thing to have in the logs constantly, since this is basically the repos in the environment variable, minus any trees that are closed. Perhaps it makes more sense to add it as a warning instead, if the two don't match?